README.md

```diff
@@ -459,7 +459,6 @@ All marks support the following style options:
 * **fill** - fill color
 * **fillOpacity** - fill opacity (a number between 0 and 1)
 * **stroke** - stroke color
-* **strokeWidth** - stroke width (in pixels)
```

Comment:
This should be kept, right? The options that are supported as channels are also listed as constant options.